### PR TITLE
VZ-5381 upgrade doc

### PR DESCRIPTION
**Pull request by pfmackin**
Document the UpgradePaused state behavior

### CODE REVIEW

**Review by stevshap — APPROVED** [review]

File: content/en/docs/setup/upgrade/_index.md [hunk 1]
```diff
@@ -176,3 +166,30 @@ NAME                     READY   STATUS    RESTARTS   AGE
 mysql-67575d8954-d4vkm   2/2     Running   0          39h
 tododomain-adminserver   4/4     Running   0          39h
 ```
+
+## Upgrade failures
+
+In Verrazzano 1.3 and later, upgrade will continue to run until it succeeds or until you delete the Verrazzano CR.  In previous versions,
+upgrade can fail and transition to the UpgradeFailed state.  If that happens, and you updated the Verrazzano Platform Operator to 1.3+,
```
Comment (rosemarymarano) [inline content/en/docs/setup/upgrade/_index.md:173]: upgrade can fail -> upgrade could fail
UpgradeFailed -> `UpgradeFailed` (code font)
Verrazzano Platform Operator -> Verrazzano platform operator

File: content/en/docs/setup/upgrade/_index.md [hunk 2]
```diff
@@ -176,3 +166,30 @@ NAME                     READY   STATUS    RESTARTS   AGE
...
+the the Verrazzano CR will transition to UpgradePaused.  To continue with the upgrade, you will need to change the CR version to the current
```
Comment (rosemarymarano) [inline content/en/docs/setup/upgrade/_index.md:174]: UpgradePaused -> `UpgradePaused`
you will need to change -> you must change

File: content/en/docs/setup/upgrade/_index.md [hunk 3]
```diff
@@ -176,3 +166,30 @@ NAME                     READY   STATUS    RESTARTS   AGE
...
+version of the Verrazzano Platform Operator.  The following steps illustrate this scenario:
```
Comment (rosemarymarano) [inline content/en/docs/setup/upgrade/_index.md:175]: Verrazzano Platform Operator -> Verrazzano platform operator

File: content/en/docs/setup/upgrade/_index.md [hunk 4]
```diff
@@ -176,3 +166,30 @@ NAME                     READY   STATUS    RESTARTS   AGE
...
+
+1. You install Verrazzano 1.1.2
```
Comment (rosemarymarano) [inline content/en/docs/setup/upgrade/_index.md:177]: Please put a period at the end of each statement.

File: content/en/docs/setup/upgrade/_index.md [hunk 5]
```diff
@@ -176,3 +166,30 @@ NAME                     READY   STATUS    RESTARTS   AGE
...
+2. You upgrad to to 1.2.0 by changing the Verrazzano CR version field to v1.2.0
```
Comment (rosemarymarano) [inline content/en/docs/setup/upgrade/_index.md:178]: upgrad to to -> upgrade to

File: content/en/docs/setup/upgrade/_index.md [hunk 6]
```diff
@@ -176,3 +166,30 @@ NAME                     READY   STATUS    RESTARTS   AGE
...
+3. Upgrade failed for some reason and the Verrazzano CR state transitions to UpgradeFailed
```
Comment (rosemarymarano) [inline content/en/docs/setup/upgrade/_index.md:179]: Upgrade  -> The upgrade
UpgradeFailed -> `UpgradeFailed`

File: content/en/docs/setup/upgrade/_index.md [hunk 7]
```diff
@@ -176,3 +166,30 @@ NAME                     READY   STATUS    RESTARTS   AGE
...
+4. You update the Verrazzano Platform Operator to 1.3.0
```
Comment (rosemarymarano) [inline content/en/docs/setup/upgrade/_index.md:180]: Verrazzano Platform Operator -> Verrazzano platform operator

File: content/en/docs/setup/upgrade/_index.md [hunk 8]
```diff
@@ -176,3 +166,30 @@ NAME                     READY   STATUS    RESTARTS   AGE
...
+5. The Verrazzano CR state transitions to UpgradePaused.
```
Comment (rosemarymarano) [inline content/en/docs/setup/upgrade/_index.md:181]: UpgradePaused -> `UpgradePaused`

File: content/en/docs/setup/upgrade/_index.md [hunk 9]
```diff
@@ -176,3 +166,30 @@ NAME                     READY   STATUS    RESTARTS   AGE
...
+6. You change the Verrazzano CR version field to v1.3.0
+7. The Verrazzano CR state transitions to Upgrading and stays in that state until it completes, then it transitions to UpgradeComplete.
```
Comment (rosemarymarano) [inline content/en/docs/setup/upgrade/_index.md:183]: Upgrading -> `Upgrading` 
UpgradeComplete -> `UpgradeComplete`

**Review by rosemarymarano — COMMENTED** [review]
Many minor text edits

File: content/en/docs/setup/upgrade/_index.md (hunk 2 above)
Comment (jmaron99) [reply content/en/docs/setup/upgrade/_index.md:174]: "the the" -> "then the"

File: content/en/docs/setup/upgrade/_index.md [hunk 10]
```diff
@@ -176,3 +166,30 @@ NAME                     READY   STATUS    RESTARTS   AGE
 mysql-67575d8954-d4vkm   2/2     Running   0          39h
 tododomain-adminserver   4/4     Running   0          39h
 ```
+
+## Upgrade failures
+
+In Verrazzano 1.3 and later, upgrade will continue to run until it succeeds or until you delete the Verrazzano CR.  In previous versions,
+upgrade could fail and transition to the `UpgradeFailed` state.  If that happens, and you updated the Verrazzano Platform Operator to 1.3+,
```
Comment (rosemarymarano) [inline content/en/docs/setup/upgrade/_index.md:173]: Verrazzano Platform Operator -> Verrazzano platform operator

File: content/en/docs/setup/upgrade/_index.md [hunk 11]
```diff
@@ -176,3 +166,30 @@ NAME                     READY   STATUS    RESTARTS   AGE
...
+then the Verrazzano CR will transition to `UpgradePaused`.  To continue with the upgrade, you must change the CR version to the current
+version of the Verrazzano platform operator.  The following steps illustrate this scenario:
+
+1. You install Verrazzano 1.1.2.
+2. You upgrade to to 1.2.0 by changing the Verrazzano CR version field to v1.2.0.
```
Comment (rosemarymarano) [inline content/en/docs/setup/upgrade/_index.md:178]: upgrade to to 1.2.0 -> upgrade to 1.2.0

**Review by rosemarymarano — COMMENTED** [review]
Just two minor edits left to do.

**Pull request merged**